frontend/pkg/frontend/frontend.go

```diff
@@ -301,7 +301,7 @@ func (f *Frontend) ArmResourceRead(writer http.ResponseWriter, request *http.Req
 	writer.WriteHeader(http.StatusOK)
 }
 
-func (f *Frontend) ArmResourceCreateOrUpdate(writer http.ResponseWriter, request *http.Request) {
+func (f *Frontend) ArmResourceCreateorUpdate(writer http.ResponseWriter, request *http.Request) {
```

Comment:
```suggestion
func (f *Frontend) ArmResourceCreateOrUpdate(writer http.ResponseWriter, request *http.Request) {
```
Any reason we renamed this?

Reply:
ah the capital O is a typo
but updates requests can come as both PUT or Patch requests so i'm splitting the func into one that handles only PATCH and one that handles both PUT/PATCH

Reply:
Yup, agreed, just keeping the old one the same :D